parsers/ENTSOE.py

```diff
@@ -127,6 +128,8 @@
 ENTSOE_EXCHANGE_DOMAIN_OVERRIDE = {
     'DE->DK-DK1': ['10Y1001A1001A63L', ENTSOE_DOMAIN_MAPPINGS['DK-DK1']],
     'DE->DK-DK2': ['10Y1001A1001A63L', ENTSOE_DOMAIN_MAPPINGS['DK-DK2']],
+    'DE->SE-SE4': [ENTSOE_DOMAIN_MAPPINGS['SE-SE4'],
```

Comment:
Any need for adding `PL->SE-SE4`, `FI->SE-SE{1,3}`and `LT->SE-SE4`?
I'm not sure the exchanges between bidding zones and countries will work.

Reply:
no need to add, it works